include/klee/Expr.h

```diff
@@ -1080,7 +1080,8 @@ class ConstantExpr : public Expr {
   }
 
   static ref<ConstantExpr> create(uint64_t v, Width w) {
-    assert(v == bits64::truncateToNBits(v, w) && "invalid constant");
+    if (w <= 64)
```

Comment:
No I meant my version of this change was this.

```c++
  static ref<ConstantExpr> create(uint64_t v, Width w) {
#ifndef NDEBUG
    if (w <= 64)
      assert(v == bits64::truncateToNBits(v, w) && "invalid constant");
#endif
    return alloc(v, w);
  }

```

It is up to you whether or not you find this tasteful.